idprovider/idprovidertest/doc.go

```diff
@@ -14,6 +14,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package contactstest provides a helper functions to create contacts file from
+// Package idprovidertest provides a helper functions to create idprovider file from
```

Comment:
```suggestion
// Package idprovidertest provides a helper functions to create ID provider file from
```
Use ID provider as two words in comments with the word `ID` in all upper case.